Ed-Fi-Exchange-OSS/EdFi-Project-Buzz
# [FIF-359] Add student assessments to API

Adds student assessment entity, service, resolver and module to the API.

## TO TEST
Migrate the database to latest
Run the sample data and confirm you have 32 student assessment records
If you run the following with studentschoolkey = 100001-1001
```json
query ($studentschoolkey: String!) {
  assessmentsbystudentschool(studentschoolkey: $studentschoolkey) {
    studentschoolkey
    assessmentidentifier
    datetaken
    score
    assessmenttitle
    studentassessmentkey
  }
}
```

## EXPECT 
When you execute the query above, it should return 

```json
{
  "data": {
    "assessmentsbystudentschool": [
      {
        "studentschoolkey": "100001-100001",
        "assessmentidentifier": "120A653A-0DD8-0085-5A54-1B6E432C4485",
        "datetaken": "1339131600000",
        "score": "1546",
        "assessmenttitle": "STAAR-EOC-English I Writing",
        "studentassessmentkey": "d7e29165-0bb5-41ed-9b5e-056629eba38a"
      },
      {
        "studentschoolkey": "100001-100001",
        "assessmentidentifier": "49A3B48E-1ABC-A87E-3DF2-6B4609AB6655",
        "datetaken": "1339131600000",
        "score": "1645",
        "assessmenttitle": "STAAR-EOC-Geometry",
        "studentassessmentkey": "6221f5af-d609-4a1b-8ab6-271d3629ee91"
      },
      {
        "studentschoolkey": "100001-100001",
        "assessmentidentifier": "6B325950-3F29-84B9-AC44-E9E42CF6817B",
        "datetaken": "1339131600000",
        "score": "1578",
        "assessmenttitle": "STAAR-EOC-Algebra I",
        "studentassessmentkey": "ba875963-732c-4222-bcc4-e0fc6957770a"
      },
      {
        "studentschoolkey": "100001-100001",
        "assessmentidentifier": "F03A9ACE-76C3-BE3A-D348-55A9B8DB6A80",
        "datetaken": "1339131600000",
        "score": "1624",
        "assessmenttitle": "STAAR-EOC-English I Reading",
        "studentassessmentkey": "62b6eb14-a721-4b1b-a019-012d8dcbf1d6"
      }
    ]
  }
}
```
